src/main/java/com/github/tomakehurst/wiremock/jetty/JettyHttpServerFactory.java

```diff
@@ -34,23 +34,15 @@ private static Constructor<? extends JettyHttpServer> getServerConstructor() {
     try {
       Class<? extends JettyHttpServer> serverClass =
           (Class<? extends JettyHttpServer>)
-              Class.forName("com.github.tomakehurst.wiremock.jetty94.Jetty94HttpServer");
+              Class.forName("com.github.tomakehurst.wiremock.jetty11.Jetty11HttpServer");
```

Comment:
I'm wondering if we should ditch the reflection now and just construct it directly.

This was originally used so that both JDK7/Jetty 9.2.x and JDK8+/Jetty 9.4.x could be selected appropriately at runtime. Since we're dropping support for all Jetty versions except 11 (for now at least) there's no good reason I can think of to load reflectively.

It'll help cold start performance too if we do it directly.

Reply:
Sure, I think direct usage will be better option

Reply:
Addressed that as well, thanks @tomakehurst